docs/source/concept_models/vivarium_sanofi_multiple_myeloma/concept_model.rst

```diff
@@ -419,14 +419,109 @@ As discussed in treatment guideline reviews, in general triplet regimens are pre
+
+We will consider the following treatment categories in our treatment model:
+
+- Isa-containing treatment regimen
+- Non-isa-containing treatment regimen
+
+.. note::
+
+	Other treatment categories to consider tracking include: reference treatment regimen from ISA RCT (polidamide + dexamethasone), daratumumab-containing regimens
+
+Importantly, we will assume that a given percentage of patients who receive isa-containing regimen for a specific line of treatment will not receive an isa-containing regimen at a later line of treatment.
+
+	Currently, we will assume that 100% of patients who receive isa-containing regimens will not receive isa-containing regimens at a later line of treatment.
+
+.. todo::
+	
+	Confirm use of probability 1.0 for ineligibility for future isa-containing treatment regimen based on past treatment with isa.
+
+	Alternative approach: make ineligible if simulant progresses on or within 60 days seven months time OR with a probability of 60% if simulant progresses within seven months. This was chosen because the ISA trial defined non-response as progression on or within 60 days or progression within 6 months after achieving at least a partial response and reported at least partial response in 60% of subjects and a median time to response among this group was 35 days).
+
+.. note::
+
+	Consider similarly assuming that patients who recieve dara-containing regimens will not receive isa-containing regimens at later lines of treatment.
+
+The overall proportion of patients who receive isa-containing treatment regimens at each line of treatment and in each scenario is shown in the table below.
+
+.. list-table:: Isatuximab treatment coverage assumptions
+   :header-rows: 1
+
+   * - Treatment line
+     - 2021 coverage (Baseline and Alternative)
+     - 2025 coverage (Baseline)
+     - 2025 coverage (Alternative)
+   * - 1st
+     - 0%
+     - 0%
+     - 20%
+   * - 2nd
+     - 0%
+     - 20%
+     - 20%
+   * - 3rd 
+     - XXX
+     - 20%
+     - 20%
+   * - 4th+
+     - XXX
+     - 20%
+     - 20%
+
+.. todo::
+
+	Describe linear scale-up
+
+	Finalize numbers currently marked as "XXX" based on commercial team input
+
+**How to assign treatment category to a simulant:**
+
+*Initialization*:
+
+	Each simulant initialized into a multiple myeloma state should be assigned to a treatment category. They should be assigned to the isa-containing treatment category or non-isa-containing treatment category with the probability of the isa-containing treatment category equal to the isatuxamib treatment coverage values for 2021 in the table above, specific to the cause model state that the simulant occupies.
+
+	For simulants assigned to the isatuxamib-containing treatment category, assign the attribute :code:`ineligible_for_future_isa = True` with a probability of 1. For all other simulants, set :code:`ineligible_for_future_isa` to :code:`False`.
```

Comment:
Under current scenario design, the probability of receiving Isa-containing regimen increases across lines of treatment, while keeping the probability of receiving non-Isa containing regimen as constant during 5 years of simulation. Notice that Pisa + Pnon-isa != 1.

Reply:
Ohh, I see. I think I may have been misunderstanding this. So that means that isa would be added to existing treatments rather than swapped out? Or just that non-isa-containing treatments would decrease in a proportional manner?

Reply:
I am confused by this. I don't see how the probability could be greater than one if it's just two binary categories where isa is or isn't part of a treatment regimen for a given line.

Reply:
How I've set it up, isa-containing treatment doesn't mean that you only get isa, it means that isa is a part of your treatment (one part of a triplet, for example).

Reply:
Yes. Ideally, the probability treated with non-isa regimen would decrease when Isa coverage increases over time. But we don't know how fast it falls down so we made an assumption of constant coverage rate for all different type of non-isa regimens. Patients now have a new option Isa, that doesn't necessarily mean they prefer tp switch to this new type of treatment.

Reply:
Hmm, I am still not following. Can we chat about this when we're free?

Reply:
Sure!